package.json

```diff
@@ -1687,10 +1687,11 @@
     "cssnano": "^5.1.12",
     "cssnano-preset-default": "^5.2.12",
     "csstype": "^3.0.2",
-    "cypress": "13.6.3",
+    "cypress": "13.15.0",
```

Comment:
Updated Cypress due to an issue where cypress tests hang locally when the browser is not focused. Reference: https://github.com/cypress-io/cypress/issues/28392 .

Reply:
For future reference, Cypress updates should be done in a different PR since there are more solutions a part from us that are using Cypress as well and they might be affected by this change.

Before merging this PR and to make sure that this is not impacting other solutions, please make sure that the `ci:all-cypress-suites` label and you update your branch with the latest main changes to trigger the execution of all the different Cypress tests of the Kibana repo.

Reply:
I'll open a separate PR with just these changes! No worries 😄

Reply:
Done: https://github.com/elastic/kibana/pull/198715, will wait on that making it in first 👍🏾